client/components/plans/plan-actions/index.jsx

```diff
@@ -194,9 +194,10 @@ module.exports = React.createClass( {
 	},
 
 	managePlanButton: function() {
+		var link = '/purchases/' + this.props.site.domain + '/' + this.props.siteSpecificPlansDetails.id;
```

Comment:
We are using `paths.purchases.managePurchase()` to create this path in `client/me/purchases`. I'm wondering if we shouldn't do the same here and all other places updated in this PR. My only concern is if shared component should depend on page specific code, i.e. `client/components/*` depend on `client/me/*`. Maybe we should move paths to `client/lib/*`?